File: Dddml.Wms.HttpServices.JavaClient/src/org/dddml/wms/domain/AttributeSetAttributeUseId.java
package org.dddml.wms.domain;


public class AttributeSetAttributeUseId
{
    private String attributeSetId;

    public String getAttributeSetId()
    {
        return this.attributeSetId;
    }

    public void setAttributeSetId(String attributeSetId)
    {
        this.attributeSetId = attributeSetId;
    }

    private String attributeId;

    public String getAttributeId()
    {
        return this.attributeId;
    }

    public void setAttributeId(String attributeId)
    {
        this.attributeId = attributeId;
    }

    AttributeSetAttributeUseId ()
    {
    }

    public AttributeSetAttributeUseId (String attributeSetId, String attributeId)
    {
        this.attributeSetId = attributeSetId;
        this.attributeId = attributeId;
    }

}

File: Dddml.Wms.AdminUI/ts/AttributeSetAttributeUseId.ts
namespace Dddml.Wms.Domain {

	export class AttributeSetAttributeUseId {

		private attributeSetId: string;

		getAttributeSetId(): string {
			return this.attributeSetId;
		}

		setAttributeSetId(attributeSetId: string) {
			this.attributeSetId = attributeSetId;
		}

		private attributeId: string;

		getAttributeId(): string {
			return this.attributeId;
		}

		setAttributeId(attributeId: string) {
			this.attributeId = attributeId;
		}

	}
}

